datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -539,7 +539,8 @@ private void validateMediumDutySdxEnablement(SdxClusterRequest sdxClusterRequest
         ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
         if (SdxClusterShape.MEDIUM_DUTY_HA.equals(sdxClusterRequest.getClusterShape())) {
             boolean mediumDutySdxEntitlementEnabled = entitlementService.mediumDutySdxEnabled(Crn.safeFromString(environment.getCreator()).getAccountId());
-            if (!mediumDutySdxEntitlementEnabled) {
+            boolean entitlementNotRequiredForCloudProvider = isMediumDutyEntitlementNotRequiredForCloudProvider(environment.getCloudPlatform());
```

Comment:
Same here. Change the function to "isMediumDutyEntitlementRequiredForCloudProvider".

Prefer using positive verbs to negative verbs.